BrainPortal/app/models/local_data_provider.rb

```diff
@@ -38,6 +38,11 @@ class LocalDataProvider < DataProvider
 
   Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
 
+  # this returns the category of the data provider -- used in view for admins
```

Comment:
I meant, explain it in the code, so that other programmer know about this convention! If it's not done, it should also be explained in the model in the base class, DataProvider.